components/brave_federated_learning/brave_federated_learning_service.cc

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_federated_learning/brave_federated_learning_service.h"
+
+#include "bat/ads/pref_names.h"
+#include "brave/components/brave_federated_learning/brave_operational_profiling.h"
+#include "brave/components/brave_federated_learning/brave_operational_profiling_features.h"
+#include "brave/components/p3a/pref_names.h"
+#include "chrome/browser/profiles/profile_manager.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/pref_service.h"
+#include "services/network/public/cpp/shared_url_loader_factory.h"
+#include "services/network/public/cpp/simple_url_loader.h"
```

Comment:
not used. Please double-check for unused includes everywhere

Reply:
my bad. Refactored this class and left there some old includes